.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+## Description
+
+Please include a summary of the changes and the related issue including the relevant motivation and context. List any dependencies that are required for this change.
+
+## Linked Issue(s)
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Enhancement (improved an existing feature)
+- [ ] This change requires a documentation update
```

Comment:
If we have to  delete options that are not relevant, I think it's better not to have check box

Reply:
I can update the text so we can simply check the appropriate option